doc/nrf/libraries/bluetooth_services/rpc.rst

```diff
@@ -7,35 +7,33 @@ Bluetooth Low Energy Remote Procedure Call
    :local:
    :depth: 2
 
-The |NCS| supports Bluetooth® Low Energy (LE) stack serialization.
-The full Bluetooth LE stack can run on another device or CPU, such as the nRF5340 DK network core using the :ref:`nrfxlib:nrf_rpc`.
+The Bluetooth® Low Energy Remote Procedure Call library allows the full Bluetooth Low Energy stack to run on another device or CPU such as the nRF5340 DK network core.
 
-Network core
-************
+Overview
+********
 
-The :ref:`ble_rpc_host` sample is designed specifically to enable the Bluetooth LE stack functionality on a remote MCU (for example, the nRF5340 network core) using the :ref:`nrfxlib:nrf_rpc`.
-You can program this sample to the network core to run standard Bluetooth Low Energy samples on the nRF5340 DK.
-You can use either the SoftDevice Controller or the Zephyr Bluetooth LE Controller for this sample.
+The library allows to use the Bluetooth Low Energy API from a different CPU or device.
+This is accomplished by running a full Bluetooth Low Energy stack on one device and serializing the API from another device.
 
-Application core
-****************
+Implementation
+==============
 
-To use the Bluetooth LE stack through nRF RPC, an additional configuration is needed.
-When building samples for the application core, enable the :kconfig:option:`CONFIG_BT_RPC_STACK` Kconfig option to run the Bluetooth LE stack on the network core.
-This option builds :ref:`ble_rpc_host` automatically as a child image.
-For more details, see :ref:`ug_nrf5340_building`.
+The library encloses the configured Bluetooth Low Energy stack and the serialization on the host (network core) side.
+The client (application core) side contains the serialization used to access the host side.
 
-Open a command prompt in the build folder of the application sample and enter the following command to build the application for the application core, with :ref:`ble_rpc_host` as child image:
+Supported backends
+==================
 
-.. code-block:: console
+The library supports the following two Bluetooth Low Energy controllers:
 
-   west build -b nrf5340dk_nrf5340_cpuapp -- -DCONFIG_BT_RPC_STACK=y
```

Comment:
Maybe, I haven't notice if this sentence was moved somewhere. Could we fit somewhere in this documentation this example how to build application with the serialized stack?

Reply:
It's in the line 85 "Set the :kconfig:option:`CONFIG_BT_RPC_STACK` Kconfig option to enable the Bluetooth RPC library."